recipe/meta.yaml

```diff
@@ -37,12 +36,14 @@ requirements:
     - matplotlib >=2.0.0
     - networkx >=1.8
     - pillow >=4.3.0
-    - dask-core >=0.15
```

Comment:
Why remove this altogether?

Reply:
Oh, in this iteration i decided that 0.9 was the correct version choice ;)